proposals/76:vaults-auctions/upgradeVaults.test.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-env node */
+import test from 'ava';
+
+import {
+  addPreexistingOracles,
+  agops,
+  ATOM_DENOM,
+  bankSend,
+  createBid,
+  getInstanceBoardId,
+  getISTBalance,
+  getLiveOffers,
+  getPriceQuote,
+  getVaultPrices,
+  openVault,
+  pushPrices,
+  USER1ADDR,
+} from '@agoric/synthetic-chain';
+import { readFile, rm } from 'node:fs/promises';
+import {
+  lastAuctionInstancePathname,
+  getDetailsMatchingVats,
+} from './vatDetails.js';
+
+const oraclesByBrand = new Map();
+
+let roundId = 2;
+
+const setupOracles = async t => {
+  const atomOutPre = await getPriceQuote('ATOM');
+  t.is(atomOutPre, '+12010000');
+
+  await addPreexistingOracles('ATOM', oraclesByBrand);
+
+  await pushPrices(11.2, 'ATOM', oraclesByBrand, roundId);
+  roundId += 1;
+};
+
+const checkNewQuotes = async t => {
+  t.log('awaiting new quotes');
+  const atomOut = await getPriceQuote('ATOM');
+  t.is(atomOut, '+11200000');
+};
+
+export const BID_OFFER_ID = 'bid-vaultUpgrade-test3';
+const createNewBid = async t => {
+  await createBid('20', USER1ADDR, BID_OFFER_ID);
+  const liveOffer = await getLiveOffers(USER1ADDR);
+  t.true(liveOffer[0].includes(BID_OFFER_ID));
```

Comment:
```suggestion
  const liveOffers = await getLiveOffers(USER1ADDR);
  t.true(liveOffers[0].includes(BID_OFFER_ID));
```

and why check `.includes(BID_OFFER_ID)` rather than `=== BID_OFFER_ID`? Does `getLiveOffers` not return an array of offer ids?

Reply:
I don't think it does. I think it's a larger structure I didn't know how to reliably untangle.